package.json

```diff
@@ -9,7 +9,7 @@
     "watch:dev": "nodemon",
     "lint": "npx eslint .",
     "lint:fix": "npx eslint --fix .",
-    "format": "npx prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
```

Comment:
was this something you removed for your individual debugging given your issues with `npx`?